doc/lsst.ap.pipe/pipeline-bps.rst

```diff
@@ -42,10 +55,35 @@ Again, as of September 2023 this assumes you are running on one of the lsst-devl
   project: ApPipe
   campaign: my_example
   
+  # The submit yaml must specify the following arguments:
+  # Default arguments provided by bps (not included here) are listed in the ctrl_bps documentation (see below).
+  payload:
+    # This will set the output collection name.
+    payloadName: my_example_name
+    # Same as -b on the command line.
+    butlerConfig: /sdf/group/rubin/repo/main/butler.yaml
+    # Same as -i on the command line; actual input collections may differ from what is shown here.
+    inCollection: u/elhoward/DM-38243/templates,DECam/defaults
+    # Same as -d on the command line. Here is an example of a small data query just for testing.
+    dataQuery: "instrument='DECam' AND skymap='decam_rings_v1' AND tract=8122"
+
+  # Add extra quantum graph options, such as specifying parameters.
+  extraQgraphOptions: "-c parameters:apdb_config=path/to/your/apdb_config.yaml"
+
+  # An example on how to add more configurations, such as clustering.
+  includeConfigs:
+    - ${AP_PIPE_DIR}/bps/clustering/clustering_ApPipe.yaml
+
+  # An example on how to customize a pipeline task.
+  pipetask:
+    # Here you can set options to various pipeline tasks if they should run with something other than the defaults you specified above.
+    subtractImages:
+      requestMemory: 4096
+
+  # Defaults that don't need to be included in your yaml unless you want to change them.
```

Comment:
I think this should be in the singular unless the following are multiple defaults, in which case maybe change the last bit from them. to them: to indicate the following is a list of optional defaults?